gateway/utils/validate.go

```diff
@@ -185,7 +185,7 @@ func Validate(where map[string]interface{}, obj interface{}) bool {
 					default:
 						return false
 					}
-
+				case "$contains":
```

Comment:
You need to write the validate logic here.

Reply:
The logic here is incorrect. You need to actually implement the contains quiet here. Think of this function as a filter